docs/topics/sb-features-and-frameworks-summary.adoc

```diff
@@ -3,10 +3,10 @@
 
 //.{SpringBoot} Features and Frameworks Summary
 
-The chapters in this guide cover the design of modern applications using link:https://projects.spring.io/spring-boot/[Spring Boot]. These concepts support development of Web or Websocket applications using either a HTTP connector or non-blocking HTTP connector.
+This guide covers the design of modern applications using link:https://projects.spring.io/spring-boot/[Spring Boot]. These concepts support developing Web or Websocket applications using either a HTTP connector or non-blocking HTTP connector.
 The applications can be packaged and deployed without modification or updated to use cloud native features on OpenShift.
 
-The features in the table below are available as a collection of missions which run on OpenShift. Some features are native as part of Kubernetes, others are available from link:https://github.com/spring-cloud-incubator/spring-cloud-kubernetes[Spring Cloud Kubernetes].
+The features in the table below are available as a collection of missions on OpenShift. Some features are native to Kubernetes, others are available from link:https://github.com/spring-cloud-incubator/spring-cloud-kubernetes[Spring Cloud Kubernetes].
```

Comment:
"missions which run on OpenShift" -> "missions on OpenShift" -- this looks wrong to me. Missions aren't part of OpenShift. You need to do some work to get them there.

Reply:
I agree with @Ladicek, the previous wording seems more on point to me.